hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -89,6 +92,9 @@ UsageAccumulator baseUsageFor(HederaFunctionality function, SubType type) {
 				return submitMessage();
 			case TokenFeeScheduleUpdate:
 				return feeScheduleUpdate();
+			default:
+				throw new IllegalArgumentException("Canonical usage unknown");
+
 		}
 
 		throw new IllegalArgumentException("Canonical usage unknown");
```

Comment:
This could be removed as default is throwing an `IllegalArgumentException`.

Reply:
@abhishek-hedera Since the function expects returning `UsageAccumulator` , without this it will have compilation issue. `default` is added to resolve code smells

Reply:
I see, thanks for the clarification.

Reply:
@abhishek-hedera has a good point. To gain both, it's simpler to do a break for `default`.